packages/foreman/rubygem-fog-vsphere/rubygem-fog-vsphere.spec

```diff
@@ -88,6 +88,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/tests
 
 %changelog
+* Tue Nov 16 2018 Chris Roberts <chrobert@redhat.com> - 2.5.0-1
```

Comment:
The date is incorrect. The 16th was a Friday. The `bump_rpm.sh` script generates a changelog entry for you and I'd recommend using that in the future.